app/bundles/CoreBundle/Views/Helper/page_actions.html.php

```diff
@@ -19,9 +19,9 @@
 }
 
 include 'action_button_helper.php';
-
-echo '<div class="std-toolbar btn-group">';
-
+if (!empty($templateButtons)) {
```

Comment:
ok - revert this if you will and I'll handle it in https://github.com/mautic/mautic/pull/2905. I want to make sure custom buttons that aren't using template buttons still have that wrapping div.